### DISCUSSION TITLE
Add GitHub Copilot Chat support to PHPStorm Plugin

### DISCUSSION THREAD

**Discussion by a-bashtannik** [community/community#50808] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

Dear GitHub Team,

I am writing to request that you add support for GitHub Copilot Chat in the PHPStorm plugin. As you are likely aware, PHPStorm is an industry-standard IDE used by many PHP developers worldwide. Additionally, GitHub Copilot has already been made available through a plugin, I personally widely use it, but I also believe that the addition of the Copilot Chat feature would be a game-changer for PHP developers and, as a PHPStorm user, I would really appreciate if such a fantastic feature become available in the plugin. 

Currently, it's available for Visual Studio and Visual Studio Code, which are, without a doubt, amazing products, but for some reason still do not fit the requirements of many PHP developers.

![image](https://user-images.githubusercontent.com/2712350/227190789-d5836936-53a9-4814-898c-99730fc25d93.png)

**Comment by Akash1134** [comment]
Hi @a-bashtannik ,

Thank you for posting in the Copilot community! We value your feedback and our team is paying attention to the feedback, feature requests & bugs we are hearing from users in the Community Discussions.

At this time, the public preview for Copilot for Chat is for VS or VSCode users. However, we'll share your feedback with the concerned team so that they can look into it.

**Reply by tudorpie** [reply]
I've been using PHPSTORM for ages now (7 years i think?). This is a deal breaker. If Github copilot X is not included in the near future, I will bail ship and embrace VS Code.

**Reply by resistdesign** [reply]
I would rather abandon Copilot and stop paying. There will be other AI solutions for IntelliJ.

**Reply by iosifnicolae2** [reply]
+1 JetBrains support

**Reply by revolutionxk** [reply]
+1 JetBrains support

**Reply by Ale1x** [reply]
+1 JetBrains support

**Reply by MrManiacc** [reply]
+1 JetBrains support

**Reply by jaweed-khan** [reply]
+1 JetBrains support

**Reply by Saud96525** [reply]
https://github.com/OAI/OpenAPI-Specification/issues/3288#issuecomment-1567222667

**Reply by zy6p** [reply]
+1 JetBrains support

**Reply by Saud96525** [reply]
6zVLxL7yyE2P3eH

**Reply by LeFou-k** [reply]
+1 Jetbrains support

**Reply by omerh2802** [reply]
+1 JetBrains support

**Reply by Khwaj7** [reply]
+1 for JetBrains support (IDEA)

**Reply by CaptainDredge** [reply]
+1

**Reply by nomercy360** [reply]
+1

**Reply by humarkx** [reply]
+1

**Reply by sgexzdkwse68792** [reply]
+1

**Reply by moeric** [reply]
+1

**Reply by frees11** [reply]
+1

**Reply by simas-b** [reply]
+1

**Reply by Kek5chen** [reply]
+1

**Reply by jbknick** [reply]
+1

**Reply by sergeByishimo** [reply]
+1

**Reply by FdvTerje** [reply]
+1

**Reply by Oranzh** [reply]
+1

**Reply by krisimmig** [reply]
+1

**Reply by s277645013** [reply]
+1

**Reply by rubanraj54** [reply]
+1

**Reply by mindrunner** [reply]
Please use reactions and subscribe to this topic instead of posting useless comments. It really pollutes inboxes of people following this thread. Thank you!

**Reply by ShamaIow** [reply]
+1

**Reply by xahon** [reply]
+1

**Reply by fjw** [reply]
+1

**Reply by iamarsenibragimov** [reply]
+100500

**Reply by inrainbowss** [reply]
+1

**Reply by lan2720** [reply]
+1000000

**Reply by crodrigues** [reply]
+1

**Reply by rogerguasch** [reply]
+1

**Reply by 021-projects** [reply]
+1 ❤️

**Reply by JDues** [reply]
+1

**Reply by healthcaremaria** [reply]
+1

**Reply by Ayid7676** [reply]
+1

**Reply by fromsomnia** [reply]
+1

**Reply by pcad-ealbrecht** [reply]
+1

**Reply by IBRAHIMDANS** [reply]
+1

**Reply by widoz** [reply]
+1 Phpstorm is one of the most used IDE in the PHP community and would make totally sense to include the chat there.

**Reply by overclokk** [reply]
+1

**Reply by NeiroNext** [reply]
+1

**Reply by samuelgfeller** [reply]
7 months later, @Akash1134 could you check with the concerned team what the status is?

It feels like you're intentionally exclusively offering this key feature only in your proprietary editor. 
We're paying a fair price for Copilot, it's poor to treat those that don't use a Microsoft IDE differently!

**Reply by HenriqueSoaresBarbosa** [reply]
+1

**Comment by bob-obringer** [comment]
Yeah... I'm not leaving WebStorm.  Would be great to have this in JetBrains tools.

**Reply by rgraffbrd** [reply]
💥

**Reply by jonathanstiansen** [reply]
I've been trying VScode again for a bit, and am thoroughly unimpressed compared to Jetbrains.

**Reply by antoniolmcandido** [reply]
really, WebStorm or PhpStorm are incredible

**Reply by Ayid7676** [reply]
I've been trying VScode again for a bit, and am thoroughly unimpressed compared to

**Reply by xtrasmal** [reply]
yeah VSCode is just a highly configurable text-editor. It lacks and is quirky compared to WebStorm and PHPStorm.

**Comment by rgraffbrd** [comment]
code whisperer is looking interesting 🤷‍♀️

**Comment by alexseif** [comment]
Yes, it will be great to have the chat feature integrated into phpstorm, either through through the existing plugin or an adjacent plugin

**Comment by danjdewhurst** [comment]
Yep, PhpStorm support would be nice thanks.

**Comment by arcticlinux** [comment]
The plugins are shared across the IntelliJ platform, PHPStorm, PyCharm, etc.  Support would be useful in all of them.

**Reply by zhuxiaoker** [reply]
+1

**Reply by fjw** [reply]
+1

**Reply by lan2720** [reply]
PyCharm +10000

**Reply by jekson** [reply]
+1

**Reply by JDues** [reply]
+1

**Reply by fromsomnia** [reply]
+1

**Comment by GiGurra** [comment]
Yeah this is really important. I use mostly intellij and goland with the regular copilot plugin, together with chatgpt on the side in the browser. Having an IDE aware capable AI model with chat is a must in the future.

I just got access to copilot chat and was really bummed to see only VS and VSCode are supported. Please add support for the jetbrains IDE family.

**Comment by chuckbenger** [comment]
+1 would love support in IntelliJ

**Comment by Jolly23** [comment]
Waiting for it
<img width="708" alt="Screenshot 2023-05-20 at 8 27 21 PM" src="https://github.com/community/community/assets/17488831/140eae0b-44bf-4584-9138-1363903e796b">

**Comment by treyreynolds** [comment]
Will also throw in my support for Jetbrains extensions for Copilot X projects

**Comment by antiftw** [comment]
+1 for support in PHPStorm / Jetbrains products.

**Comment by cholinkol** [comment]
+1 PHPStorm / Jetbrains

**Comment by JiriPetruzelkaSW** [comment]
Even now there are AI solutions but I don't think there's anything on the level of Copilot X

Imho it's a big deal and it would be in PHPStorm's best interest to adopt it as soon as it is possible

**Reply by a-bashtannik** [reply]
I tried codeium, but return back. Copilot is an industry-standard today.

**Comment by stewartcelani** [comment]
+1 support for Copilot X in all Jetbrains products (Rider specifically here). Just received access and bummed I'll have to use VS.

**Comment by powernic** [comment]
+1 PHPStorm / Jetbrains

**Comment by beautiful-sea** [comment]
+1 PHPStorm / Jetbrains

**Comment by azn13** [comment]
+1 for all IntelliJ products

**Comment by MarcEspiard** [comment]
+1 PHPStorm / Jetbrains

**Comment by joshuaneedham** [comment]
It's not JetBrains pushing the Copilot Chat here it's GitHub and Microsoft. It is a nice feature to have in VS Code however code snippets frequently get 'filtered' as it says so it's still really a work in progress even on VS Code. I do believe once they get all the kinks worked out it will be an amazing addition. 

As for AWS Codewhisper it's ok as well but doesn't support languages such as ERB for example and being a Ruby/Rails engineer that is a problem.

+1 for JetBrains integration.

I just purchased IntelliJ Ultimate and having to go back to VS Code to test out CoPilot Chat is a bit frustrating but worth it to help advance the technology.

**Comment by xahon** [comment]
+1 for JB

**Comment by wantedrbk** [comment]
+1 JetBrains really needs it , even for beta

**Comment by SteveChurch** [comment]
+1 Jetbrains support.

**Comment by mbertoluci** [comment]
+1 JetBrains products, please 🙏🏻

**Comment by iosifnicolae2** [comment]
+1 JetBrains support

**Comment by tiaden** [comment]
+1 for all IntelliJ products

**Comment by leiyangyou** [comment]
Wondering how hard would it be to reverse engineer the vscode plugin and reimplement it for intellij (with the help of copilot & chatgpt)

**Comment by lejsue** [comment]
+1 for phpatorm/JB

**Comment by LanDeQuHuXi** [comment]
+1 and mark this thread

**Comment by Ale1x** [comment]
+1 For Copilot in phpstorm

**Reply by sospopo** [reply]
+1 For Copilot in goland

**Comment by faycal1** [comment]
+1 For Copilot in phpstorm

**Reply by basiltt** [reply]
+1 JetBrains tools like PyCharm and WebStorm support

**Comment by frasko** [comment]
+1 For Copilot Chat in PHPStorm

**Comment by omerh2802** [comment]
+1 JetBrains tools like PyCharm and WebStorm support

**Comment by jporti3** [comment]
+ 1 Jetbrains tools, I would love to have that chat in my favorite Java IDE

**Comment by edcarandang** [comment]
+1 for Jetbrains tools.

**Comment by siegfried0111** [comment]
+1 Jetbrains

**Comment by alexisljn** [comment]
+1 Jetbrains support

**Reply by riggerz29** [reply]
+1 for copilot with jetbrains

**Comment by stenyin** [comment]
+1 Jetbrains

**Comment by ChenTechDev** [comment]
+1 web storm

**Comment by ZihangYin-Compass** [comment]
+1 Jetbrains

**Comment by crutcher** [comment]
+1 for generic JetBrains plugin support for this; the existing github copilot is amazing, but switching IDEs is an expensive request

**Comment by rubanraj54** [comment]
+1 for phpstorm

**Comment by sgexzdkwse68792** [comment]
+1 Jetbrains support

**Comment by LordOfTheRats** [comment]
+1 JetBrains

**Comment by Securiteru** [comment]
+1 Jetbrains

**Comment by qingyanbaby** [comment]
+1 Jetbrains

**Comment by frees11** [comment]
+1 Jetbrains

**Comment by t-works** [comment]
+1 Jetbrains

**Comment by sai3563** [comment]
+1 Jetbrains

**Reply by macrombilux** [reply]
> +1 Jetbrains

+1 Macrombilux

**Comment by invertibleMatrix** [comment]
+1 for Jetbrains Rider

**Comment by jporti3** [comment]
Hello, Do you have any news from this?
Thanks  @Akash1134

**Comment by zh-plus** [comment]
+1 Jetbrains

**Comment by dawiddyd** [comment]
+1 Jetbrains

**Comment by eyetractive** [comment]
+1 Jetbrains

**Comment by hansfpc** [comment]
+1 Jetbrains

**Comment by sergeByishimo** [comment]
+1 Intellij

**Comment by islamsaka** [comment]
+1 Jetbrains

**Comment by valentinwmtz** [comment]
+1 Jetbrains

**Comment by 0xWeverton** [comment]
+1 Jetbrains

**Comment by ningblue** [comment]
+1 Jetbrains

**Comment by chrisreedio** [comment]
+1 Jetbrains.

I use both VSCode and PHPStorm and have access to Copilot Chat via VSCode Insiders and would love to also be able to use it in PHPStorm/Jetbrains tools.

Thanks!

**Reply by 0kn0t** [reply]
+1

**Comment by bassem97** [comment]
+1 jetbrains !!!

**Reply by 1saifj** [reply]
+1 jetbrains !!!

**Comment by 1saifj** [comment]
+1 jetbrains !!!

**Comment by pcariel** [comment]
+1 jetbrains !!!

**Comment by mrsinham** [comment]
+1 jetbrains

**Comment by En3Tho** [comment]
+1 JetBrains!

**Comment by bassem97** [comment]
1+ jetbrains

**Comment by Freeshnik** [comment]
+1 JetBrains!

**Comment by ragevox** [comment]
Looking forward for Webstorm

**Comment by Frisbeetarian** [comment]
webstorm

**Comment by mindrunner** [comment]
Please use reactions and subscribe to this topic instead of posting useless comments. It really pollutes inboxes of people following this thread. Thank you!

**Comment by tysont** [comment]
+1! Can we get an update?

**Comment by Akash1134** [comment]
Hi everyone, Copilot Chat is still in its public preview stage and we're collecting your valuable feedback based on the VS and VSCode IDEs. We do not have an ETA for supporting other IDEs right now but we'll update you when we have details to share as we're working on it. Thank you for your patience and understanding.

**Comment by luisjose-al** [comment]
+1 Jetbrains tools

**Comment by supanadit** [comment]
+1 Jetbrains Tools

**Comment by DionTech** [comment]
+1 JetBrains!

**Comment by mightymatth** [comment]
+1 jetbrains 😄  

any news? 👀

**Reply by mindrunner** [reply]
![image](https://github.com/community/community/assets/1413542/3e87e4aa-a8b1-4d02-b05f-868a762661a3)


@everyone here, please stop those annoying +1 messages here. Especially if you did not read the thread. Nothing here will speed up the release of such plugin. It just spams everyones inboxes :(

**Reply by fjw** [reply]
I disagree. It gives a signal what we as customers think and need.

If your inbox is spammed, turn of notifications. Don't tell us what to say or not say!

**Reply by mindrunner** [reply]
I am not telling people, but rather asked for the favor of using the reactions feature for what it is made for. There is absolutely no benefit in having Threads full of nonsense +1 posts. 

It is obvious, we all want this feature as well as it's obvious that it'll come when they have it ready. 

A GitHub bug report is just not the right place to start a demonstration.

I would definitely be the first one enabling the feature of only getting a notification for useful posts if GitHub has it available.

**Reply by Frisbeetarian** [reply]
What a useless comment, just turn off notifications if it bothers you.

**Comment by zerubeus** [comment]
why don't Jetbrain use make a gpt-4 plugin and take things in hands?

**Reply by maicol07** [reply]
Actually, there is: https://blog.jetbrains.com/idea/2023/06/ai-assistant-in-jetbrains-ides/

**Reply by zerubeus** [reply]
Nothing compared to copilot really, they fell far behind, I don't understand how they didn't see this comming, copilot was around way before even chatgpt became a thing, and at Jetbrean they are sleeping deep... understandable when compared to Microsoft budged wise.

**Comment by Exlll** [comment]
GitHub should provide a public API for Copilot so the community (or Jetbrains) can implement these plugins themselves.

**Reply by zerubeus** [reply]
GitHub using OpenIA for to create plugins for Microsoft IDE's (VS, VSC) why do you think Gihub should do the work of JetBrains for them?, they can just do the same and use OpenAI to create plugins!

**Reply by xahon** [reply]
@zerubeus
1. Extending the number of users (additional money)
2. They already did regular Copilot plugin for JB

I don't see any problem for them to create a plugin

**Reply by Exlll** [reply]
@zerubeus Copilot is [marketed](https://github.com/features/copilot) as being available for JetBrains IDEs, Neovim, and VS (Code). If Github is unable or unwilling to add proper support for the full feature set of Copilot to other IDEs besides VS (Code), then a public (paid) Copilot API would be helpful to those people/companies who want to implement that missing functionality themselves and who can add support for new Copilot features in a timely manner. Besides, that would also allow the use of Copilot's code completion functionality in other tools without having to rely on one of those four aforementioned IDEs.

If Github instead wants to tie Copilot to Microsoft products (which is understandable, since they are owned by it), then perhaps they should just say so, so that people who use other IDEs can move on instead of having to have these discussions and wait for new features that may never be added to the IDE they are using.

**Comment by kowalk** [comment]
jetbrains +1

**Comment by mariojackson** [comment]
+1 for JetBrains tools

**Comment by iholder101** [comment]
+1 jetbrains!

**Comment by herbertroth** [comment]
+1 for JetBrains

**Comment by Assclo** [comment]
After being compromised think  I'll need visual assistance
Make or offer any suy to help

**Comment by pierrexp9** [comment]
+1 for JetBrains

**Comment by roke22** [comment]
+1 for JetBrains

**Comment by jekson** [comment]
+1 phpstorm

**Comment by vilnitsky** [comment]
+1 phpstorm

**Comment by gollum23** [comment]
+1 for Jetbrains

**Comment by TahseenAlaa** [comment]
Dears any update for phpStorm to support chat with Copilot?

**Comment by Fendo181** [comment]
+1 jetbrains! 🙏

**Comment by cesarve77** [comment]
+999999999 for Jetbrains

**Reply by zerubeus** [reply]
JetBrains has its AI chat now, and it is based on gpt4 and doing better than copilot chat for JetBrains products, this request is not relevant anymore. It is free if you already have a License.

**Reply by tudorpie** [reply]
Indeed, it's even better than Copilot. Kudos to them

**Reply by moeric** [reply]
Where/how can we find this new AI chat?

**Reply by janw-me** [reply]
I think it's this
https://blog.jetbrains.com/idea/2023/06/ai-assistant-in-jetbrains-ides/

**Reply by zerubeus** [reply]
Open your Jetbrain IDE, go to settings > Tools > AI Assistant, login with your Jetbrain account, and voilà, you have access to AI chat, and commit generated by AI and many other AI futures, and most important a much better Model than Copilot, Jetbrain AI combine OpenAI gpt4 and other LLM models developed by Jetbrain for their ides, enjoy

PS: the commit generation features is a killer :)

**Reply by moeric** [reply]
I'm using PhpStorm version 2023.2.1 and there is no "AI Assistant" under settings > Tools, I guess they haven't add it to PhpStorm yet, I hope they do it soon. Thanks for the info anyway.

**Reply by zerubeus** [reply]
Did you download AI assistant from plugin marketplace ?

**Reply by zerubeus** [reply]
Ahh I saw there's a waiting list.

**Reply by zerubeus** [reply]
But it is available in phpstorm https://blog.jetbrains.com/phpstorm/2023/06/ai-features-in-phpstorm-phpstorm-2023-2-eap-5/ if you still don't see it, download the EAP version here

**Reply by diegovogel** [reply]
There's still a waiting list even with the EAP. I've been on it for 2 months now 🫠

**Comment by Gompje** [comment]
Got the mail yesterday I got access to copilot chat beta. 💃 🎉 Maybe I this will be waaay  better than ChatGPT 3.5. 😃 this will be sooo great! 

No plug-in for my main IDE: phpstorm. 😭 Cannot stand vscode for some reason. Does not work for me nor my workflow.

No web interface either... 🤔 

Oh well. 🤷‍♀️ 
That's it then. 

Maybe I'm missing out on something great. I will not know. 

I'm on the waiting list for JetBrains AI. Maybe that will go through soon. 
Cannot wait. 
Hope rises.
😉 

+1 for Phpstorm support. 

This is imho a must is your goal is gathering data during this test phase. Like others said it's the standard in the php community. And I'm not switching out on a good tool just to test this potential one feature.

+1 for a website version that works on the GitHub website, to fill that void. App support would be awesome too.

-99 now I've got the feeling I'm paying for features I cannot uses, even if this came "for free". Maybe I should put my monies somewhere else..

// but as a dev I do understand priorities must be set.

**Reply by NeiroNext** [reply]
Waiting this functional for JetBrains 😔

**Comment by philepsybo** [comment]
Full jetbrains support is a must in my opinion for IDE-plugins. Out of the IDEs i have tried over the years, no other dev-tools seemed to be even close to that degree of maturity and efficiency once you know how to use them properly. 

It is highly unlikely that developers who are used to that maturity will switch to another IDE to make one plugin completely functional as intended if that means that a bunch of their workflows loose 90% of their efficiency.

I loved using "normal" copilot funcitonality so far and I learned a lot...it's a great product!
Even pushed it so all our devs do now get a license by default.
But I really dont see myself leaving jetbrains just to get the upcoming additional features working for me.

**Comment by elokus** [comment]
+1 PyCharm

**Comment by daniel-wozniak** [comment]
+1 PhpStorm

**Comment by andergsx** [comment]
+1 phpStorm

**Comment by Solunsky** [comment]
+1 phpStorm

**Comment by NeiroNext** [comment]
+1 phpStorm

**Comment by NeiroNext** [comment]
PHPStorm one of the most popular IDE why we need so long time, why don't add this functional for two IDE at time? It's some sad :c

**Comment by upmorpheus** [comment]
Yes, it will be great to have the chat feature integrated into phpstorm, either through through the existing plugin or an adjacent plugin

**Comment by xblabs** [comment]
Any update here?

**Comment by a-bashtannik** [comment]
I suggest you try Jetbrains AI Assistant - https://blog.jetbrains.com/blog/2023/10/12/ai-assistant-update-october-2023/ - I already tested, and it looks promising. 

MS seems to be forcing us to use their product in such a manner, by luring us in their ecosystem.

**Reply by michabbb** [reply]
@a-bashtannik funny! how? it´s a closed beta !

**Reply by a-bashtannik** [reply]
@michabbb I use [EAP versions](https://www.jetbrains.com/resources/eap/) and Jetbrains granted me access, not sure if it is closed or not.

**Reply by michabbb** [reply]
enjoy it. i hope others get access soon.... 😞

**Reply by zerubeus** [reply]
it is even better that copilot, it generates commit message for me LOL :D

**Comment by dbuurstede** [comment]
+1 phpStorm

**Comment by Akash1134** [comment, marked as answer]
👋  We are happy to announce that a private beta of GitHub Copilot Chat is now available for users of JetBrains IDEs, including IntelliJ, PyCharm, WebStorm, Android Studio, and more. 🚀 

**This private beta is available to Copilot Business customers and Copilot Individual users.**

### To get access to the private beta, [sign up for this waitlist](https://github.com/github-copilot/chat_jetbrains_waitlist_signup/join).

GitHub Enterprise Cloud (GHEC) administrators interested in participating in the private beta should reach out to your GitHub account manager or [contact our sales team](https://github.com/enterprise/contact) to make the feature available for your enterprise.

We’d love your feedback on this new release. Please [use this link to share your feedback](https://gh.io/copilot-chat-jb-feedback) or ideas on how to improve the product. Thank you for your patience and support.

**Reply by SananShahverdi** [reply]
Hello, currently I am facing problem regarding accepting my application for the waitlist. I have applied for that 2 weeks ago but still no answer. Can you please make it clear of what is approximate waiting time.

Thanks in advance.

**Reply by erictompkins** [reply]
@Akash1134 I have Copilot business, I've enabled chat in the Copilot settings in Github, and I've installed the Copilot plugin in PHPStorm. However, I don't have access to Copilot Chat in PHPStorm. It says I need to join the waitlist, but the waitlist is only for individuals. How does a business get added to the waitlist or get access to the beta?

**Reply by PiotrPress** [reply]
I have the same issue with PHPStorm :(

**Reply by theCroh** [reply]
Same problem here.

**Reply by realjjaveweb** [reply]
When will the true release be? I don't see where the hold is since basic copilot in-code is already there, adding a window with req/resp should  be a 2-week effort release included if not shorter?

**Reply by mariusz-andrzejewski-droptica** [reply]
I had the same feeling about this. Didn't want to say it out loud, but seems like this might be prolonged intentionally, for example, to attract the developers to the other software. I don't get it, I believe there is a huge demand on the market to have the copilot chat in the JetBrains ecosystem, this should have been released a long long time ago.

**Comment by upmorpheus** [comment]
Thanks for your valuable suggestions.

These are really helpful!

**Comment by mariusz-andrzejewski-droptica** [comment]
+1 phpStorm
BTW is this going to be something in addition to AI Assistant that is developed by JetBrains?

**Comment by rmpel** [comment]
Hello.
I hope it's ok I am replying this to @Akash1134 since this topic is answered by you.

I was wondering when chat will be available in phpStorm for corporate users.
I can use chat (but not inline copilot) in a code-with-me session, but still it is not working in "normal" phpStorm mode.

I have mentioned this somewhere many months back, but to this day, the panel still gives "click here to join the waitinglist". I am pretty sure we are already accepted as our administration department already (again, many months back) approved usage.

As I said back then; not a complaint, and happy to wait, but the wait is getting to be a very long one.

Thanks.

**Reply by realjjaveweb** [reply]
Chat (icon on panel on right in the new UI) is definitely working in the "normal" version of phpstorm now - but you have to **update the plugin** and **restart the IDE**. However it is true I am still missing the **inline chat** :cry:

**Comment by rippersvk** [comment]
tried to add copilot plugin with chat to PHPStorm last week and it still only offers chance to be part of beta...
We really want that chat!

**Reply by rippersvk** [reply]
it works !

**Reply by realjjaveweb** [reply]
For others - make sure you've updated the plugin(s) to the latest version and restarted your IDE :slightly_smiling_face:

**Comment by johnrclark3** [comment]
I have GitHub copilot plugin and it is authorized but when I click `Open GitHub Copilot Chat` it does nothing.

**Reply by realjjaveweb** [reply]
Have you tried restarting the IDE? (and made sure you've updated the plugin to the latest?)
